FastReport.Base/Utils/Config.cs

```diff
@@ -523,22 +534,45 @@ private static void LoadPlugins()
 
         private static void LoadPluginsInCurrentFolder()
         {
+            string searchPattern = "FastReport.Data.*.dll";
+            if (string.IsNullOrWhiteSpace(ApplicationFolder) || (Directory.Exists(ApplicationFolder) == false))
+            {
+#if DEBUG                
+                Console.WriteLine($"Config ApplicationFolder ({ApplicationFolder}) is empty or invalid... can not load Plugins {searchPattern}");
+#endif                
+                return;
+            }
             // find all plugin-connector in current directory
-            var plugins = Directory.GetFiles(ApplicationFolder, "FastReport.Data.*.dll");
+            var plugins = Directory.GetFiles(ApplicationFolder, searchPattern);
 
             // initialize
             foreach(var pluginName in plugins)
             {
                 try
                 {
+#if DEBUG
+                    Console.WriteLine($"try to load plugin {pluginName}");
+#endif                    
+#if NETCOREAPP
```

Comment:
ok i can remove this because i do not need this. i only want to remove the exception at line

var plugins = Directory.GetFiles(ApplicationFolder, searchPattern);

because ApplicationFolder is empty because AppDomain.CurrentDomain.BaseDirectory is empty

at the moment i have no database source, json, excel, .... i have a two DataSets read from xml Files

ok more in the issue create a new one